grails-app/services/org/pih/warehouse/tableroapi/IndicatorDataService.groovy

```diff
@@ -314,15 +314,22 @@ class IndicatorDataService {
         Date date = LocalDate.now().minusMonths(querySize).toDate()
 
         def query = ReceiptItem.executeQuery("""
-            select s.shipmentNumber as number, s.name as name, count(ri.id) as count, s.requisition as requisition 
+            select 
+                s.shipmentNumber, 
+                s.name, 
+                count(ri.id), 
+                s.requisition
```

Comment:
Do we need this?

Reply:
Also you aliased these in the previous version. Did you mean to remove the alias?

Reply:
`s.requisition` is used to build the link between each table's item and it's stock movement page.
Link is composed by: "/openboxes/stockMovement/show/" + `s.requisition` 

Concerning the alias, it was used in the beginning when the request was built on dataService, but got useless on RecepitItem and we forgot to delete before.

Reply:
Ok, see if you can use s.requisition.id instead. s.requisition alone implicitly joins requisition which ends up creating an N+1 query on picklist.